README.md

```diff
@@ -1,53 +1,74 @@
+The [Run MATLAB Command](#run-matlab-command) GitHub&reg; action enables you to execute a MATLAB&reg; script, function, or statement on a [GitHub-hosted](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners) or [self-hosted](https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners) runner:
 
-MATLAB exits with exit code 0 if the specified script, function, or statement
-executes successfully without error. Otherwise, MATLAB terminates with a nonzero
-exit code, which causes the build to fail. You can use the
-[`assert`](https://www.mathworks.com/help/matlab/ref/assert.html) or
-[`error`](https://www.mathworks.com/help/matlab/ref/assert.html) functions in
-the command to ensure that builds fail when necessary. When you use this task,
-all of the required files must be on the MATLAB search path.
+- If you want to use a self-hosted runner, you must set up a computer with MATLAB (R2013b or later) as your runner. The action uses the first MATLAB version on the runner's path.
 
-**Note**: By running the code in this action, you will be executing third-party
-code that is licensed under separate terms.
+- If you want to use a GitHub-hosted runner, you must include the [Set Up MATLAB](https://github.com/matlab-actions/setup-matlab/) action in your workflow to install MATLAB on the runner. Currently, this action is available only for public projects and does not include transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;.
 
-## Usage
+## Usage Examples
+Use the **Run MATLAB Command** action to run MATLAB scripts, functions, and statements tailored to your specific needs. You can use this action to flexibly customize your test run or add a MATLAB-related step to your workflow. 
 
-You can use this action `with`:
-| Argument  | Description |
-|-----------|-------------|
-| `command` | (Required) Script, function, or statement to execute. <br/> If the value of `command` is the name of a MATLAB script or function, do not specify the file extension. If you specify more than one MATLAB command, use a comma or semicolon to separate the commands. <br/> **Example**: `myscript` <br/> **Example**: `results = runtests, assertSuccess(results);`
+### Run MATLAB Script on Self-Hosted Runner
+Use a self-hosted runner to run the commands in a file named `myscript.m` in the root of your repository.
 
-### Using a GitHub-hosted runner?
-If you are using a GitHub-hosted runner, you can use the [Set up MATLAB action](https://github.com/matlab-actions/setup-matlab/) to install MATLAB on the runner.
+```yaml
+name: Run MATLAB Script on Self-Hosted Runner
+on: [push]
+jobs:
+  my-job:
+    name: Run MATLAB Script
+    runs-on: self-hosted
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Run script
+        uses: matlab-actions/run-command@v0
+        with:
+          command: 'myscript'
```

Comment:
We may want to consider not showing the command wrapped in single quotes. I believe these quotes are [generally unneeded](https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml) and a user may want to use single quotes in their actual MATLAB command, in which case they would need to learn how to properly escape them.

Reply:
Good point. Thanks.